Documentation/ceph-cluster-crd.md

```diff
@@ -151,6 +151,14 @@ For more details on the mons and when to choose a number other than `3`, see the
 * `removeOSDsIfOutAndSafeToRemove`: If `true` the operator will remove the OSDs that are down and whose data has been restored to other OSDs. In Ceph terms, the osds are `out` and `safe-to-destroy` when then would be removed.
 * `cleanupPolicy`: The section for confirming that cluster data should be forcibly deleted. The cleanupPolicy should only be added to the cluster when the cluster is about to be deleted. After any field of the cleanup policy is set, Rook will stop configuring the cluster as if the cluster is about to be destroyed in order to prevent these settings from being deployed unintentionally.
   * `deleteDataDirOnHosts`: If `yes-really-destroy-data` the operator will automatically delete the hostpath of cluster nodes when a `delete cephcluster` command is issued. Only `yes-really-destroy-data` and an empty string are valid values for this field.
+  * sanitizeDisks: sanitizeDisks represents whether we should sanitize OSD disks on cluster deletion
```

Comment:
I'd like to close on the design questions [here](https://github.com/rook/rook/pull/5452#discussion_r428309392) before we go ahead with any new properties in the CR.

Reply:
We know we will have a `sanitizeDisks` spec right? What goes in it yes is still TBD, but I think we can go with it. What's blocking?

Reply:
Updated based on our discussion [here](https://github.com/rook/rook/pull/5452#discussion_r430591750)

Reply:
done